packages/ffe-accordion/less/ffe-accordion.less

```diff
@@ -2,111 +2,117 @@
     width: 100%;
     border: 1px solid transparent;
     font-family: 'SpareBank1-regular', 'MuseoSans-500', arial, sans-serif;
+}
 
-    &--focus {
-        border-color: @ffe-farge-vann;
+.ffe-accordion-item {
+    margin-bottom: @ffe-spacing-xs;
+    transition: margin-bottom @ffe-transition-duration @ffe-ease-in-out-back;
+    box-shadow: 0 1px 4px rgb(38 38 38 / 30%);
+    border-radius: 16px;
+    border: 2px solid transparent;
+    background-color: @ffe-farge-hvit;
+
+    &__heading {
+        margin: 0;
+    }
+
+    &__heading-button {
+        width: 100%;
+        background: none;
+        border: none;
+        padding: @ffe-spacing-sm;
+        font-family: 'SpareBank1-medium', 'MuseoSansRounded-700', arial,
+            sans-serif;
+        cursor: pointer;
+        outline: none;
+        color: @ffe-farge-vann;
+        transition: all @ffe-transition-duration @ffe-ease;
```

Comment:
Denne ser ikke ut til å ha noen funksjon per nå, for å få animert knappen på hover må transition ligge der verdiene som animeres ligger. Altså `.ffe-accordion-item__heading-icon-wrapper` (`background`) og  `.ffe-accordion-item__heading-icon` (`fill`).